src/custom/state/user/reducer.ts

```diff
@@ -102,6 +102,8 @@ function _initialStatePerChain(chainId: number) {
   )
 }
 
+const ALL_SUPPORTED_CHAIN_IDS = [SupportedChainId.MAINNET, SupportedChainId.GNOSIS_CHAIN, SupportedChainId.GOERLI]
```

Comment:
no longer exposed from the sdk?

Reply:
Will add this to SDK in the next version

Reply:
👍